kalite/control_panel/views.py

```diff
@@ -407,7 +407,9 @@ def account_management(request):
             # Never report this error; don't want this logging to block other functionality.
             logging.error("Failed to update student userlog activity: %s" % e)
 
-    return student_view_context(request)
+    c = student_view_context(request)
+    c['restricted'] = settings.DISABLE_SELF_ADMIN
```

Comment:
It's a replacement of a removed template context processor that set the context var 'restricted'... but it was ONLY used in this particular place so I put it directly in the view and removed the context processor overkill.